src/components/common/headers/Header/Header.style.ts

```diff
@@ -0,0 +1,46 @@
+import { Theme } from '@emotion/react';
+import { css } from '@emotion/react';
```

Comment:
```suggestion
import { css, Theme } from '@emotion/react';
```

이렇게 하나로 합쳐줘도 될 것 같습니다!